pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [project]
 name = "rs-client-libraries"
-version = "0.0.0" # see: https://github.com/mtkennerly/poetry-dynamic-versioning
+version = "0.0.2" # see: https://github.com/mtkennerly/poetry-dynamic-versioning
```

Comment:
You should set the version back to 0.0.0 here. We put it to 0.0.2 only for a test. The real version will be set automatically by the ci/cd.